Ginger/GingerCoreCommon/ALM/PublishToALMConfig.cs

```diff
@@ -16,13 +16,30 @@
 */
 #endregion
 
+using System;
 using System.ComponentModel;
 using Amdocs.Ginger.Common;
+using Amdocs.Ginger.Repository;
+using static GingerCoreNET.ALMLib.ALMIntegrationEnums;
 
 namespace GingerCore.ALM
 {
-    public class PublishToALMConfig
+    public class PublishToALMConfig : INotifyPropertyChanged
```

Comment:
what is this class for? (we have 'RunSetActionPublishToQC.cs' for below fields. no?)

Reply:
has a lot of reference , won't change the logic